docs/docs/Get-Started/get-started-quickstart.md

```diff
@@ -4,188 +4,502 @@ slug: /get-started-quickstart
-3. Ask the bot: `Who won the Oscar in 2024 for best movie?`
-4. The bot's response should be similar to this:
+  </TabItem>
+  <TabItem value="JavaScript" label="JavaScript">
+
+```js
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+const url = 'http://localhost:7860/api/v1/run/29deb764-af3f-4d7d-94a0-47491ed241d6';
+
+// Store the previous answer
+let previousAnswer = null;
+
+async function askAgent(question) {
+    const payload = {
+        "output_type": "chat",
+        "input_type": "chat",
+        "input_value": question
+    };
+
+    const options = {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(payload)
+    };
+
+    try {
+        const response = await fetch(url, options);
+        const data = await response.json();
+        
+        // Extract the message from the nested response
+        const message = data.outputs[0].outputs[0].outputs.message.message;
+        return message;
+    } catch (error) {
+        return `Error: ${error.message}`;
+    }
+}
+
+async function startChat() {
+    console.log("\nAsk the agent anything, such as 'What is 15 * 7?' or 'What is the capital of France?'");
+    console.log("Type 'quit' to exit or 'compare' to see the previous answer");
+
+    const askQuestion = () => {
+        rl.question('\nYour question: ', async (userQuestion) => {
+            if (userQuestion.toLowerCase() === 'quit') {
+                rl.close();
+                return;
+            }
+
+            if (userQuestion.toLowerCase() === 'compare') {
+                if (previousAnswer) {
+                    console.log(`\nPrevious answer was: ${previousAnswer}`);
+                } else {
+                    console.log("\nNo previous answer to compare with!");
+                }
+                askQuestion();
+                return;
+            }
+
+            const result = await askAgent(userQuestion);
+            console.log(`\nAgent's answer: ${result}`);
+            previousAnswer = result;
+            askQuestion();
+        });
+    };
+
+    askQuestion();
+}
+
+startChat(); 
+```
 
-```text
-The Oscar for Best Picture in 2024 was awarded to "Oppenheimer,"
-produced by Emma Thomas, Charles Roven, and Christopher Nolan.
+  </TabItem>
+</Tabs>
+
+## Apply temporary overrides to a flow run
+
+To change a flow's values at runtime, Langflow offers **Temporary Overrides**, also called **Tweaks**.
+Tweaks are added to the API request to the `/run` endpoint, and temporarily change component parameters within your flow.
+They don't modify the underlying flow configuration or persist between runs.
+
+The easiest way to modify your request with tweaks to the `/run` endpoint is in the **API access** pane, in the **Input schema** pane.
+
+1. In the **Input schema** pane, select the parameter you want to modify in your next request.
```

Comment:
Please revise these steps:

- Explain how to get to the input schema pane from the visual editor.
- Line 478 is confusing.
- Line 479 is numbered but it's not an instruction, it's just an example.
- Step 3/Line 498 can be merged with lines 480/481. You can simplify the guidance to something  to the effect of "Langflow updates the `tweaks` object in the code snippets based on your input parameters. Use the updated code snippets in your script to run your flow with your overrides." I don't think we need to tell them specifically to execute the code or something. This section is more like "what now", giving a hint at how to use tweaks in general.